### ISSUE TITLE
[A11yAudit] Inputs relating to user information will be able to be programmatically determined [I]

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1333]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-8 Inputs - Password and Phone
The purpose of inputs relating to user information cannot be programmatically determined

ISSUE DESCRIPTION
Completing forms online, particularly if these forms require the user to accurately recall and enter personal information, can be difficult for users with cognitive and learning disabilities. Ensuring that input and content purpose can be programmatically determined means that additional tools/assistive technologies have the potential to aid users to better understand and operate web content. For instance, in the context of login/account registration processes, programmatically identifiable form controls allow tools such as password managers to prefill/autofill certain known fields, which can greatly aid users with cognitive/memory issues.

Many of the form controls in this sample are generic and do not explicitly relate to user information. However, both the password input (samples 43-45) and phone input (samples 49-51) specifically collect user information, but their purpose cannot be programmatically determined.

RECOMMENDATIONS
Currently, for web content, the most commonly accepted solution for this success criterion is to ensure that known/common inputs (such as home/business address, email, phone number, and any other common Input Purposes for User Interface Components relating to personal user information) are identified explicitly in markup using relevant autocomplete values from HTML 5.2. This enables user agents and tools (such as browsers, password managers, and assistive technologies) to provide useful functionality such as auto-filling fields with stored personal information (meaning users don't have to remember / enter this information manually).

**AC**
- Password and Phone should have default values for the autocomplete attribute, maybe also the ability to set custom values.
